docs/src/pages/[platform]/components/view/react.mdx

```diff
@@ -1,94 +1,85 @@
 import { View } from '@aws-amplify/ui-react';
 import { ViewDemo } from './demo';
-import { Example } from '@/components/Example';
-
-View is a container that contains stuff. View is the most abstract component on top of which all other components live.
+import { Example, ExampleCode } from '@/components/Example';
+import {
+  AccessibilityExample,
+  CustomAttributesExample,
+  DefaultViewExample,
+  DisabledStateExample,
+  SetRenderedHTML,
+  StylePropsExample,
+} from './examples';
 
 ## Demo
 
 <ViewDemo />
 
 ## Usage
 
-Import the View component and styles.
+Import the View component.
 
-```jsx
-import { View } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+<Example>
+  <DefaultViewExample />
+  <ExampleCode>
+```jsx file=./examples/DefaultViewExample.tsx
 
-<View>Nice view! 🏔</View>;
 ```
-
-<Example>
-  <View>Nice view! 🏔</View>
+  </ExampleCode>
 </Example>
 
 ### Set rendered HTML element
 
-Change the HTML element rendered by View with the `as` property.
+To change the HTML element rendered by the View, use the `as` prop.
+
+<Example>
+  <SetRenderedHTML />
+  <ExampleCode>
+```jsx file=./examples/SetRenderedHTML.tsx
 
-```jsx
-<View as="section">I'm a Section</View>
 ```
 
-<Example>
-  <View as="section">I'm a Section</View>
+  </ExampleCode>
 </Example>
 
 ### Disabled state
 
-```jsx
-<View as="button" isDisabled={true} onClick={() => alert("This won't fire 🚫")}>
-  You can't click me!
-</View>
-```
+To mark a View as disabled, pass the `isDisabled` prop.
 
 <Example>
-  <View
-    as="button"
-    isDisabled={true}
-    onClick={() => alert("This won't fire 🚫")}
-  >
-    You can't click me!
-  </View>
+  <DisabledStateExample />
+  <ExampleCode>
+```jsx file=./examples/DisabledStateExample.tsx
+
+```
+
+  </ExampleCode>
 </Example>
 
 ### Accessibility
 
-Setting an `aria-label` attribute and `role` for an icon button
+The example below demonstrates setting an `aria-label` attribute and `role` for an icon button.
+
+<Example>
+  <AccessibilityExample />
+  <ExampleCode>
+```jsx file=./examples/AccessibilityExample.tsx
 
-```jsx
-<View ariaLabel="To the moon!" role="button" onClick={() => alert('👋 hello')}>
-  🚀
-</View>
 ```
 
-<Example>
-  <View
-    ariaLabel="To the moon!"
-    role="button"
-    onClick={() => alert('👋 hello')}
-  >
-    🚀
-  </View>
+  </ExampleCode>
 </Example>
 
-### Custom attributes
+### Standard HTML attributes
 
-Pass any attribute to the underlying HTML element.
-
-```jsx
-<View as="progress" data-progress-bar max="100" value="50" />
-```
+View will accept any of the standard HTML attributes that its underlying HTML element accepts. View renders a `<div>` by default.
```

Comment:
Would it make sense to mention that `<View>` renders a `div` by default at the top of the page? To me, this is the attribute that makes the `<View>` component what it is so I'd expect that to be called out within the first few lines of the docs.